epochStart/metachain/epochStartData.go

```diff
@@ -210,11 +208,8 @@ func (e *epochStartData) lastFinalizedFirstPendingListHeadersForShard(shardHdr *
 		prevShardHdr, err := process.GetShardHeader(currentHdr.GetPrevHash(), e.dataPool.Headers(), e.marshalizer, e.store)
 		if err != nil {
 			go e.requestHandler.RequestShardHeader(currentHdr.ShardID, currentHdr.GetPrevHash())
-			if e.epochStartTrigger.Epoch()-currentHdr.GetEpoch() > maxEpochDifference {
-				log.Warn("shard remained in an epoch that is too old", "shardID", currentHdr.ShardID, "shard Epoch", currentHdr.Epoch, "meta Epoch", e.epochStartTrigger.Epoch())
-				break
-			}
-			return nil, nil, nil, err
+			log.Warn("shard remained in an epoch that is too old", "shardID", currentHdr.ShardID, "shard Epoch", currentHdr.Epoch, "meta Epoch", e.epochStartTrigger.Epoch())
```

Comment:
Is it ok to leave this warn? It seems unrelated now. Why not logging also the real error err

Reply:
the real error is that it does not find the header.

Reply:
so the message "shard remained in an epoch that is too old" is relevant?